db/migrate/20201008132106_add_avatar_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvatarToUsers < ActiveRecord::Migration[6.0]
```

Comment:
why do you need this extra migration?
use this one `db/migrate/20201007140046_create_users.rb` 
the same for index

Reply:
What should I do? I think so:
1. Rollback these 3 migrations(adding index, adding avatar, create user)
2. Correct db/migrate/20201007140046_create_users.rb
3. rails db:migrate
... but i can't rollback migration for creating user, because it is Irreversible
Can i do 1 extra migration for adding avatar and index?

Reply:
locally you can drop the `users` table through the MySQL console and clear the `schema_migrations` table then run `db:migrate`
it's not so good to create a lot of migrations with changes in the same table in one PR

Reply:
actually, you can compare all of your migration to one
add only comment to fields from the devise gem

Reply:
How should I clear schema_migrations table?

Reply:
Done